src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -580,8 +588,8 @@ class ZaakEigenschapViewSet(
     required_scopes = {
         "list": SCOPE_ZAKEN_ALLES_LEZEN,
         "retrieve": SCOPE_ZAKEN_ALLES_LEZEN,
-        "create": SCOPE_ZAKEN_BIJWERKEN,
-        "destroy": SCOPE_ZAKEN_BIJWERKEN,
+        "create": SCOPE_ZAKEN_BIJWERKEN | SCOPE_ZAKEN_GEFORCEERD_BIJWERKEN,
```

Comment:
This will no doubt cause a patch-version change to the API spec.

Shouldn't we generate the API spec as well as part of this PR?

Reply:
Done